graylog2-server/src/main/java/org/graylog2/rest/resources/system/contentpacks/CatalogResource.java

```diff
@@ -25,6 +26,7 @@
 import org.graylog2.audit.jersey.NoAuditEvent;
 import org.graylog2.contentpacks.ContentPackService;
 import org.graylog2.contentpacks.model.entities.EntitiesWithConstraints;
```

Comment:
It looks like the `EntitiesWithConstraints` class is only referenced in some unused imports in `ContentPackService` and `CatalogResource` and can be removed.